samples/Server/src/routes/createRoom.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { getResourceConnectionString } from '../lib/envHelper';
+import { RoomsClient, CreateRoomOptions } from '@azure/communication-rooms';
+import { createUser } from '../lib/identityClient';
+
+const router = express.Router();
+
+/**
+ * route: /createRoom/
+ *
+ * purpose: Create a new room.
+ *
+ * @returns The new roomId as string
+ */
+
+router.post('/', async function (req, res, next) {
+  // create RoomsClient
+  const roomsClient: RoomsClient = new RoomsClient(getResourceConnectionString());
+
+  const validFrom = new Date();
+  const validUntil = new Date(validFrom.getTime() + 5 * 60 * 1000);
+
+  const user = await createUser();
+
+  // Options payload to create a room
+  const createRoomOptions: CreateRoomOptions = {
+    validFrom: validFrom,
+    validUntil: validUntil,
+    participants: [
+      {
+        id: user,
+        role: 'Attendee'
```

Comment:
can we pass the role up as a post  body parameter so we can set our role correctly and not assume everyone is an attendee? I think we can definitely default to attendee if no role is provided.

Reply:
Actually we don't need to pass in participants to create a room. Removed the participant option. Thanks.